configs/resnet_strikes_back/metafile.yml

```diff
@@ -0,0 +1,78 @@
+
+Collections:
+  - Name: ResNet strikes back
+    Metadata:
+      Training Data: COCO
```

Comment:
The "In Collection" needs to be "Faster R-CNN". Only "Algorithm" has a corresponding "Collections".